### PR TITLE
Docs: update users API

### PR DESCRIPTION
**Pull request by papagian**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by oddlittlebird — APPROVED** [review]
Thanks, Sofia!

**Comment by oddlittlebird** [comment]
You might want to have another backend expert check it for code's sake, but from a doc perspective it looks good to me.

File: docs/sources/http_api/user.md [hunk 1]
```diff
@@ -38,14 +38,22 @@ Content-Type: application/json
     "name": "Admin",
     "login": "admin",
     "email": "admin@mygraf.com",
-    "isAdmin": true
+    "isAdmin": true,
+    "isDisabled": false,
+    "lastSeenAt": "2020-04-10T20:29:27+03:00",
+    "lastSeenAtAge': "2m",
+    "authLabels": ["OAuth"]}
```
Comment (marefr) [inline docs/sources/http_api/user.md:45]:
```suggestion
    "authLabels": ["OAuth"]
```

File: docs/sources/http_api/user.md [hunk 2]
```diff
@@ -38,14 +38,22 @@ Content-Type: application/json
...
     "id": 2,
     "name": "User",
     "login": "user",
     "email": "user@mygraf.com",
-    "isAdmin": false
+    "isAdmin": false,
+    "isDisabled": false,
+    "lastSeenAt": "2020-01-24T12:38:47+02:00",
+    "lastSeenAtAge": "2M",
+    "authLabels": []}
```
Comment (marefr) [inline docs/sources/http_api/user.md:56]:
```suggestion
    "authLabels": []
```

File: docs/sources/http_api/user.md [hunk 3]
```diff
@@ -80,14 +88,22 @@ Content-Type: application/json
       "name": "Admin",
       "login": "admin",
       "email": "admin@mygraf.com",
-      "isAdmin": true
+      "isAdmin": true,
+      "isDisabled": false,
+      "lastSeenAt": "2020-04-10T20:29:27+03:00",
+      "lastSeenAtAge': "2m",
+      "authLabels": ["OAuth"]},
```
Comment (marefr) [inline docs/sources/http_api/user.md:95]:
```suggestion
      "authLabels": ["OAuth"]
```

File: docs/sources/http_api/user.md [hunk 4]
```diff
@@ -80,14 +88,22 @@ Content-Type: application/json
...
       "id": 2,
       "name": "User",
       "login": "user",
       "email": "user@mygraf.com",
-      "isAdmin": false
+      "isAdmin": false,
+      "isDisabled": false,
+      "lastSeenAt": "2020-01-24T12:38:47+02:00",
+      "lastSeenAtAge": "2M",
+      "authLabels": []}
```
Comment (marefr) [inline docs/sources/http_api/user.md:106]:
```suggestion
      "authLabels": []
```

**Review by marefr — APPROVED** [review]
LGTM, but added some suggestions of fixes

**Pull request merged**